### ISSUE TITLE
0.10rc1: `import` crashes on postgres `numeric` column

### ISSUE THREAD

**Issue by craigds** [koordinates/kart#442]
Did an import from this table:

```
 owner           | character varying |           |          |
 agency          | character varying |           |          |
 name            | character varying |           |          |
 usage           | character varying |           |          |
 parktype        | character varying |           |          |
 number_fac      | bigint            |           |          |
 area            | numeric           |           |          |
 len             | numeric           |           |          |
 GEOMETRY        | geometry          |           |          |
```

It crashed with:

```
Traceback (most recent call last):
  File "kart_cli.py", line 4, in <module>
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/cli.py", line 334, in entrypoint
  File "site-packages/click/core.py", line 829, in __call__
  File "site-packages/click/core.py", line 782, in main
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/cli.py", line 158, in invoke
  File "site-packages/click/core.py", line 1259, in invoke
  File "site-packages/click/core.py", line 1066, in invoke
  File "site-packages/click/core.py", line 610, in invoke
  File "site-packages/click/decorators.py", line 21, in new_func
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/init.py", line 340, in import_
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/fast_import.py", line 395, in fast_import_tables
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/dataset3.py", line 388, in import_iter_feature_blobs
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/pk_generation.py", line 237, in features
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/schema.py", line 334, in hash_feature
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/schema.py", line 327, in encode_feature
  File "site-packages/kart-0.10.0rc1+ci.2385.git5cb60f3b-py3.7.egg/kart/serialise_util.py", line 40, in msg_pack
  File "site-packages/msgpack/__init__.py", line 46, in packb
  File "msgpack/_packer.pyx", line 284, in msgpack._cmsgpack.Packer.pack
  File "msgpack/_packer.pyx", line 290, in msgpack._cmsgpack.Packer.pack
  File "msgpack/_packer.pyx", line 287, in msgpack._cmsgpack.Packer.pack
  File "msgpack/_packer.pyx", line 263, in msgpack._cmsgpack.Packer._pack
  File "msgpack/_packer.pyx", line 263, in msgpack._cmsgpack.Packer._pack
  File "msgpack/_packer.pyx", line 281, in msgpack._cmsgpack.Packer._pack
TypeError: can not serialize 'decimal.Decimal' object
```

**Comment by olsen232** [comment]
I had an adapter to fix this in psycogp2 times. I took it out to see if it was still needed when I recently ported everything to a consistent sqlalchemy layer, instead of a bit of a mix of everything - it wasn't on my system. Not sure why it's not consistent, but I'll put the adapter back

**Comment by olsen232** [comment]
Slightly misdiagnosed - the adapter to fix this *was* still needed when I ported everything to use adapters in the sqlalchemy layer (as opposed to the psycopg2 layer). But, I thought it wasn't, because, at some point I broke the test that checks that all datatypes can be roundtripped through all database types losslessly (or "approximately" losslessly) - ironically it looks like I roundtripped some types in a lossy way at some point and then checked the results into test data, meaning, certain types were no longer being tested.

**Comment by craigds** [comment]
Looks like this was fixed in 710adde5119efb5ef24295ce10bf41695556fdea
